packages/fast-development-site-react/src/components/design-system/index.ts

```diff
@@ -10,8 +10,8 @@ export interface IDevSiteDesignSystem {
 const devSiteDesignSystemDefaults: IDevSiteDesignSystem = {
     foregroundColor: "#000",
     backgroundColor: "#FFF",
-    brandColor: "#0078D4",
-    lightGrey: "#EEEEEE",
+    brandColor: "#343434",
+    lightGrey: "#F4F5F6",
```

Comment:
Will this color be for all text values or simply for certain instances of text? 

I think we'll want a foreground and background color similar to what we have in the msft styles to set font color and canvas background color. I'm not sure that we want to be setting globals for any color values on it. I think we'll eventually want to remove this as a global and instead manipulate the global color values to accommodate this. 

I don't know that we need to *immediately* make the change above, but we should consider it as part of a follow up